config/routes.rb

```diff
@@ -13,4 +13,6 @@
   get '/auth/:provider/callback', to: 'sessions#create'
   get '/sign_out', to: 'sessions#destroy', as: :sign_out
   get '/style_guide', to: 'styleguide#index'
+  # routes for august only pages
```

Comment:
reckon this comment is redundant as your commit message does a good job of explaining the temporary nature of the work. ideally comments should only be used in the code in extreme circumstances, as they tend to go out of date and linger long past their usefulness.